content/blog/2022/2022-01-16-spring-boot-null-safety-annotations.md

```diff
@@ -0,0 +1,152 @@
+---
+authors: [tom]
```

Comment:
Please add yourself in the file data/authors.yml and then put your name here so that the article is attributed to you